vmdb/app/controllers/application_controller.rb

```diff
@@ -1027,18 +1016,16 @@ def get_reports_menu(group=session[:group], tree_type="reports", mode="menu")
     else
       # Building custom reports array for super_admin/admin roles, it doesnt show up on menu if their menu was set which didnt contain custom folder in it
       temp = Array.new
-      subfolder = Array.new
-      rep = Array.new
-      @custom_folder = Array.new
+      subfolder = %w{ Custom }
+      @custom_folder = [ @sb[:grp_title] ]
       @custom_folder.push(@sb[:grp_title])
```

Comment:
@tenderlove Is this line still needed?  Won't `@custom_folder` have 2 elements with `@sb[:grp_title]` as a result?